gems/pending/util/miq-ipmi.rb

```diff
@@ -8,60 +8,48 @@
 
 require 'util/runcmd'
 require 'util/miq-extensions'
+require 'rubyipmi'
 
 class MiqIPMI
   def initialize(server=nil, username=nil, password=nil)
     @server = server
     @username = username
     @password = password
-    @status = self.chassis_status rescue nil
+    @connection = Rubyipmi.connect(@username, @password, @server, "ipmitool")
+    @status = chassis_status
     @vendor = nil
   end
 
   def connected?
-    !@status.nil?
+    !!@status
   end
 
   def power_state
-    result = run_command("chassis power status")
-    result.split(" ").last
+    @connection.chassis.power.status
   end
 
   def power_on
-    run_command("chassis power on")
+    @connection.chassis.power.on
   end
 
   def power_off
-    run_command("chassis power off")
+    @connection.chassis.power.off
   end
 
   def power_reset
-    if self.power_state == "off"
-      self.power_on
-    else
-      self.power_state_change('reset')
-    end
-  end
-
-  def power_state_change(new_state)
-    #status, on, off, cycle, reset, diag, soft
-    run_command("chassis power #{new_state}")
+    @connection.chassis.power.reset
   end
 
   def chassis_status
-    parse_key_value("chassis status")
-  end
-
-  def lan_info
```

Comment:
I don't see it being used anywhere, so I didn't see any reason to keep it in our wrapper class.